WordPressKit/RemotePost.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+import ObjectiveC
+
+@objcMembers public class RemotePost: NSObject {
+
+    public static let statusDraft = "draft"
+    public static let statusPending = "pending"
+    public static let statusPrivate = "private"
+    public static let statusPublish = "publish"
+    public static let statusScheduled = "future"
+    public static let statusTrash = "trash"
+    public static let statusDeleted = "deleted" // Returned by wpcom REST API when a post is permanently deleted.
```

Comment:
I was going to suggest to define a `Status` `enum`, but I guess that wouldn't work because of the `@objcMembers` requirement.

Reply:
Although...  I guess we could define the `enum` in an extension and use it here in the assignment?

```swift
extension RemotePost {

  enum Status: String {
     case draft
     case pending
     ...

...

public static let statusDraft = Status.draft.rawValue
public static let statusPending = Status.pending.rawValue
...
```

Reply:
Yes, definitely. There are lots of space to make the translated code "more Swift". I'm currently looking into the model's parsing code and in turn may change their property declarations, which kind of relate to your proposal here.